gravitee-apim-console-webui/src/management/settings/top-apis/migrated/top-apis.component.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { EMPTY, Observable, Subject } from 'rxjs';
+import { catchError, filter, switchMap, takeUntil, tap } from 'rxjs/operators';
+import { sortBy } from 'lodash';
+import { GIO_DIALOG_WIDTH, GioConfirmDialogComponent, GioConfirmDialogData } from '@gravitee/ui-particles-angular';
+
+import { TopApi } from './top-apis.model';
+import { TopApisDialogComponent } from './top-apis-dialog/top-apis-dialog.component';
+
+import { SnackBarService } from '../../../../services-ngx/snack-bar.service';
+import { TopApiService } from '../../../../services-ngx/top-api.service';
+import { Api } from '../../../../entities/api';
+
+@Component({
+  selector: 'app-top-apis',
+  templateUrl: './top-apis.component.html',
+  styleUrls: ['./top-apis.component.scss'],
+})
+export class TopApisComponent implements OnInit, OnDestroy {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
```

Comment:
```suggestion
  private unsubscribe$: Subject<void> = new Subject<void>();
```
FIY : Personal question 😁  , I prefer this because we're only interested in the event here. A true or false boolean is not necessary.